src/m365/entra/commands/m365group/m365group-user-set.ts

```diff
@@ -42,28 +45,44 @@ class EntraM365GroupUserSetCommand extends GraphCommand {
     this.#initOptions();
     this.#initValidators();
     this.#initOptionSets();
+    this.#initTypes();
   }
 
   #initTelemetry(): void {
     this.telemetry.push((args: CommandArgs) => {
       Object.assign(this.telemetryProperties, {
         teamId: typeof args.options.teamId !== 'undefined',
+        teamName: typeof args.options.teamName !== 'undefined',
         groupId: typeof args.options.groupId !== 'undefined',
-        role: args.options.role
+        groupName: typeof args.options.groupName !== 'undefined',
+        ids: typeof args.options.ids !== 'undefined',
+        userNames: typeof args.options.userNames !== 'undefined'
       });
     });
   }
 
   #initOptions(): void {
```

Comment:
Not a part of your issue, but could you please replace all the double quotes with single quotes?